docs/Guides/Digital_Ocean.md

```diff
@@ -14,18 +14,18 @@
 4. Select the repository that contains your source code, and click Next.
 
 <p align="center">
-    <img src="./AppSource.png" alt="Logo">
+    <img src="../AppSource.png" alt="Logo">
```

Comment:
Put all the images inside an `assets` folder please

Reply:
Sir, I have created the assets folder as requested but I have to traverse back to the previous directory because if we do not do so it will search the images in `/Guides/Digital_Ocean/assets/CreateApp.png` which does not exist.  By traversing back to one level lower it will search in `/Guides/assets/CreateApp.png`  which is the true path as per the structure.